### DISCUSSION TITLE
I can't seem to modify the teams

### DISCUSSION THREAD

**Discussion by thepian** [community/community#37083] [Repositories]
I have given direct access to a github user to one of my repositories, but I can't find where to remove it. It seems that the link to the direct access member should be the place to change it, but nothing happens when you click the link.

<img width="583" alt="Screenshot 2022-10-24 at 09 30 36" src="https://user-images.githubusercontent.com/72638/197472145-929d325a-4890-4509-a699-c70c84de0ec5.png">

**Comment by radar-blip** [comment]
Maybe this would help: https://docs.github.com/en/enterprise-server@3.3/admin/user-management/managing-organizations-in-your-enterprise/removing-users-from-teams-and-organizations

**Comment by ossanna16** [comment]
Hi there @thepian and welcome to our community! Thank you for asking a great question 🙂
